src/components/staffing/ProjectCell.js

```diff
@@ -1,24 +1,48 @@
 import React from 'react'
 import { Cell } from 'fixed-data-table'
+import { IGNORED_PROJECT_NAMES } from '../../constants/ignoredProjectNames'
 
 export default class ProjectCell extends React.Component {
-
   static propTypes = {
     data: React.PropTypes.array.isRequired,
     field: React.PropTypes.string,
     rowIndex: React.PropTypes.number,
     onClick: React.PropTypes.func,
-  }
+  };
 
   render() {
     const { rowIndex, field, data, ...props } = this.props
+    const sortedWeeks = Object.keys(data[rowIndex].staffing)
+    sortedWeeks.sort()
+    const currentWeek = sortedWeeks[0]
+    const projects = data[rowIndex].projects
+      ? data[rowIndex].projects.filter(
+          (project) => { return !IGNORED_PROJECT_NAMES.includes(project) }
+        )
+      : []
+    const currentProjectStaffing = data[rowIndex].staffing[currentWeek]
+      ? data[rowIndex].staffing[currentWeek]
+      : []
+    const currentProjects = projects.filter(
+      (project) => { return currentProjectStaffing[project] > 0 }
+    )
+    const upcomingProjects = projects.filter(
+      (project) => { return !currentProjects.includes(project) }
+    )
+    const currentProjectsText = currentProjects.join(', ')
+    const projectsConnectingText = currentProjects.length > 0 && upcomingProjects.length > 0 ? ', ' : ''
+    const upcomingProjectsText = upcomingProjects.join(', ')
+    const individualProjectText = data[rowIndex].projects ? '' : data[rowIndex][field]
```

Comment:
I'm not sure I understand what this represents 🤔

Reply:
Since the same component is used to display information about the person, and any individual projects they're on, the solution this repo uses is that different types of object can be passed into the component. If displaying a user (top of image), the object has a `projects` property, and if displaying a project (bottom of image), the object has a `project` property. This line  displays the individual project name if the `projects` property does not exist.
<img width="527" alt="image" src="https://user-images.githubusercontent.com/113339746/211003672-4a0bb2ae-100a-4693-8e7c-0a3cb0c736fb.png">
I'm not the biggest fan of this solution, but doing it more neatly would probably involve a fairly large rework.

Reply:
Oh interesting 🤔  Does it mean that we never have:
- <b>{currentProjectsText}</b> + {separator} + {upcomingProjectsText}
- {individualProjectText}

At the same time? If so we could clarify that in the code!

Reply:
That's correct. Updated the code to make this more clear!